airflow/timetables/interval.py

```diff
@@ -33,22 +33,14 @@
 Delta = Union[datetime.timedelta, relativedelta]
 
 
-class _DataIntervalTimetable(Timetable):
+class _DataIntervalTimeTable(Timetable):
```

Comment:
We standardise on _timetable_ as a word so the capitalisation change should be reverted.